server/src/workspace/utils/get-resolver-name.util.ts

```diff
@@ -21,6 +21,10 @@ export const getResolverName = (
       return `update${pascalCase(objectMetadata.nameSingular)}`;
     case 'deleteOne':
       return `delete${pascalCase(objectMetadata.nameSingular)}`;
+    case 'updateMany':
+      return `updateMany${pascalCase(objectMetadata.namePlural)}`;
```

Comment:
@gitstart-twenty Juste use the name plural without the `Many`

```suggestion
      return `update${pascalCase(objectMetadata.namePlural)}`;
```

Reply:
Okay